### ISSUE TITLE
I have written an article about SQLLEX

### ISSUE THREAD

**Issue by kingabzpro** [v1a0/sqllex#11]
# I hope you 💖it. @v1a0 

[SQLLEX-Simple-and-Faster](https://deepnote.com/@abid/SQLLEX-Simple-and-Faster-7WXrco0hRXaqvAiXo8QJBQ#)

**Comment by kingabzpro** [comment]
Also, I have been promoting your project on my community at Deepnote [Deepnote](https://community.deepnote.com/c/learning/new-technology-is-in-town-sqllex)

**Comment by v1a0** [comment]
Dear kingabzpro, thank you so much for your article, parts "CSV to SQLLEX.db"  and "Testing on Available SQLITE3x Database" just amazing examples of sqllex usage possibilities. 
If you don't mind I would like to add this parts to sqllex wiki, of course I'll mention you as author of examples and add link to your article.

Let me know if you'll need any help with sqllex.

# I💖it! Thanks again!

**Comment by kingabzpro** [comment]
Yes, you can use it, you can also add my article link if they want to read it all. other than that, do let me know if you want anything else. My community loves your project. I will be doing Data analysis using SQLLEX next and then compare it with duck.db .
